### ISSUE TITLE
`cln_grpc`: How to use default behavior of `exposeprivatechannels`?

### ISSUE THREAD

**Issue by ErikDeSmedt** [ElementsProject/lightning#7601]
# Context

The `invoice`-method in the JSON-rpc has a `exposeprivatechannels`-field.
If this field is not specified Core Lightning will only include private channels in the invoice if no public channels are available.

I'd like to get the same behavior using `cln_grpc`. In this case the `exposeprivatechannels`-field which expects a vector of short-channel-id's. If the vector is empty, the invoice will not contain any private channels.

**Comment by daywalker90** [comment]
I don't believe the behaviour is different using cln-(g)rpc. I think it's just a bit random or depending on your test setup if the invoice contains any channels.
